types/ros2d/tslint.json

```diff
@@ -0,0 +1,79 @@
+{
```

Comment:
Please enable all the rules that are disabled here. New packages should rarely need to disable rules.